requests/packages/__init__.py

```diff
@@ -34,9 +34,3 @@
 except ImportError:
     import chardet
     sys.modules['%s.chardet' % __name__] = chardet
-
-try:
-    from . import idna
-except ImportError:
-    import idna
-    sys.modules['%s.idna' % __name__] = idna
```

Comment:
@Lukasa we do still need this. That said, I think we can hide it in a function performs the import though.